src/components/gateway/GatewayCreatePage.tsx

```
@@ -0,0 +1,1452 @@
+        ? {
+            groupVersionKind: RESOURCES.Gateway.gvk,
+            name: nameEdit,
+            namespace: namespaceEdit,
+            isList: false,
+          }
+        : null,
+    [nameEdit, namespaceEdit],
+  );
+
+  const [gatewayData, gatewayLoaded, gatewayError] =
+    useK8sWatchResource(gatewayWatchResource);
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)